owncloudApp/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -166,7 +166,7 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public void onAttach(Context context) {
         super.onAttach(context);
-        Log_OC.e(TAG, "onAttach");
+        Log_OC.v(TAG, "onAttach");
```

Comment:
Maybe you used this to test the different log modes? Can you keep it as it was before? Thanks

Reply:
This makes a lot of noise ! If you see a red logcat and nothing is wrong, then it's senseless